v0/destinations/iterable/transform.js

```diff
@@ -352,11 +382,34 @@ function batchEvents(arrayChunks) {
 
     // Batch event into dest batch structure
     chunk.forEach(ev => {
+      if (chunk[0].message.operation === "catalogs") {
+        // body will be in the format:
+        //   {
+        //     "documents": {
+        //         "test-1-item": {
+        //             "abc": "TestValue"
+        //         },
+        //         "test-2-item": {
+        //             "abc": "TestValue1"
+        //         }
+        //     },
+        //     "replaceUploadedFieldsOnly": true
+        // }
+        batchCatalogResponseList.documents[
+          ev.message.endpoint.split("/").pop()
```

Comment:
Rather than relying on end point we should use some identifier at `ev`

Reply:
we are using identifier option. This endpoint is required to create the endpoint for batch.